.github/workflows/ossf-scorecard.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
+name: Scorecard supply-chain security
```

Comment:
Let's follow the same naming like the other workflows.
So it should be `Security - Scorecard`